src/main/java/hudson/plugins/throttleconcurrents/ThrottleJobProperty.java

```diff
@@ -62,6 +62,12 @@
     private transient boolean throttleConfiguration;
     private @CheckForNull ThrottleMatrixProjectOptions matrixOptions;
 
+    // The paramsToUseForLimit is assigned by end-user configuration and
+    // is generally a string with names of build arguments to consider,
+    // and is empty, or has one arg name, or a token-separated list of
+    // such names (see PARAMS_LIMIT_SEPARATOR) below.
```

Comment:
The `PARAMS_LIMIT_SEPARATOR` field from #59 isn't being introduced yet in this PR, so can we remove the reference from the comment?

Reply:
ack